### PR TITLE
Cauchy distribution

### PR DESCRIPTION
**Pull request by eggrobin**
Fix the "crashes crash" bug (due to adding 0-part vessels to the physics bubble), the bug where the plugin would still manage the physics bubble at ground level on bodies without an atmosphere, and the bug where a crashed vessel would keep being simulated.
Improved performance by not calling `FindObjectOfType` at every frame.

### CODE REVIEW

File: ksp_plugin/plugin.hpp [hunk 1]
```diff
@@ -210,6 +210,7 @@ class Plugin {
   // already in |next_physics_bubble_->parts|.
   virtual void AddVesselToNextPhysicsBubble(GUID const& vessel_guid,
                                             std::vector<IdAndOwnedPart> parts);
+  virtual bool PhysicsBubbleIsEmpty() const;
```
Comment (pleroy) [inline ksp_plugin/plugin.hpp]: Comment, unglue.

File: ksp_plugin_test/interface_test.cpp [hunk 2]
```diff
@@ -386,6 +389,16 @@ TEST_F(InterfaceTest, PhysicsBubble) {
   XYZ const velocity =
       principia__BubbleVelocityCorrection(plugin_.get(), kParentIndex);
   EXPECT_THAT(velocity, Eq(XYZ{66, 55, 44}));
+
+  EXPECT_CALL(*plugin_, PhysicsBubbleIsEmpty()).WillOnce(Return(true));
+  bool const empty = principia__PhysicsBubbleIsEmpty(plugin_.get());
+  EXPECT_THAT(empty, Eq(true));
```
Comment (pleroy) [inline ksp_plugin_test/interface_test.cpp]: `EXPECT_TRUE`

File: ksp_plugin/plugin.hpp (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin/plugin.hpp]: Done.

File: ksp_plugin_test/interface_test.cpp (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_test/interface_test.cpp]: Done.

**Pull request merged**